bin/delayed_job

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/home/bottlenose/.rbenv/shims/ruby
```

Comment:
Sounds like we're missing a step in the install instructions to correctly set up rvm / PATH. The same problem will likely show up with any scripts.